### PR TITLE
Reduce size of logo in readme

### PR DESCRIPTION
**Pull request by htor**
i really like the logo, but i think it doesn't really need to be that huge. most people (me included) just want to get down to the actual docs. 
thanks for making this great testing library! it feels really good!

### CODE REVIEW

**Comment by sindresorhus** [comment]
Don't do changes to `media/header.png`.

File: readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# [![AVA](media/header.png)](https://ava.li)
+<img src="media/header.png" title="ava" alt="ava logo" width="530px">
```
Comment (sindresorhus) [inline readme.md]: You have not preserved the `title from what it was.

Comment (sindresorhus) [inline readme.md]:
```suggestion
<img src="media/header.png" title="AVA" alt="AVA logo" width="530">
```

**Comment by htor** [comment]
@sindresorhus 
i think i've covered everything

**Comment by novemberborn** [comment]
Thanks @htor!

**Pull request merged**